py3status/module.py

```diff
@@ -657,6 +657,16 @@ def deprecation_log(item):
             if not hasattr(self.module_class, "py3"):
                 setattr(self.module_class, "py3", Py3(self))
 
+            # Subscribe to udev events if on_udev_* dynamic variables are
+            # configured on the module
+            on_udev_vars = [
+                i for i in dir(self.module_class) if i.startswith("on_udev_")
+            ]
+            for on_udev_var in on_udev_vars:
+                on, udev, subsystem = on_udev_var.split("_")
```

Comment:
Pyudev could have some snake_case variables. This prevents `too many values to unpack` (if any).
```suggestion
                subsystem = on_udev_var[8:]
```